auction/src/components/Controls/SelectField.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import classNames from 'classnames';
+
+const SelectField = props => {
+    const { name, id, options, fullWidth, onChange } = props;
+    const classes = classNames('select-field', {'full-width': fullWidth});
+
+    return (
+        <select onChange={onChange} className={classes} name={name} id={id}>
+            {options.map((option, index) => <option value={index}>{option}</option>)}
```

Comment:
`{
    options.map((option, index) => {
       <option value={index}>{option}</option>)
    }
}` and move `<option value={index}>{option}</option>` to new line